src/RenderWebGL.js

```diff
@@ -805,17 +809,11 @@ class RenderWebGL extends EventEmitter {
      * @returns {int} The ID of the topmost Drawable under the picking location, or
      * RenderConstants.ID_NONE if there is no Drawable at that location.
      */
-    pick (centerX, centerY, touchWidth, touchHeight, candidateIDs) {
+    pick (centerX, centerY, touchWidth, touchHeight, candidateIDs = this._visibleDrawList) {
         const gl = this._gl;
 
         touchWidth = touchWidth || 1;
         touchHeight = touchHeight || 1;
-        const candidateIDsWereProvided = !!candidateIDs;
-        candidateIDs = candidateIDs || (this._drawList.filter(id => {
-            const drawable = this._allDrawables[id];
-            const uniforms = drawable.getUniforms();
-            return drawable.getVisible() && uniforms.u_ghost !== 0;
```

Comment:
this no longer checks the "ghost" for the pick - is the non-ghosty bit here intentional? all other touching tests generally "count" touching a ghost 100 pixel